cdci_data_analysis/flask_app/dispatcher_query.py

```diff
@@ -547,20 +545,25 @@ def resolve_job_url(self):
         # what if scw list from request overwrites that in self.par_dict?
         self.set_scws_related_params(request)
 
-        self.validate_job_id(request_parameters_from_scratch_dir=True)
+        self.validate_job_id(request_parameters_from_scratch_dir=True, job_resolution=True)
 
         return self.generate_products_url_from_par_dict(
             self.app.config['conf'].products_url,
             self.par_dic
             )
 
-    def validate_job_id(self, request_parameters_from_scratch_dir=False):
+    def validate_job_id(self, request_parameters_from_scratch_dir=False, job_resolution=False):
         """    
         makes sure self.job_id is compatible, for given user token, with:
         if request_parameters_from_scratch_dir:
             parameters found in directory matching job_id
         else: 
             parameters provided in this request
+
+        if job_resolution:
```

Comment:
this is no more, yes?

Reply:
indeed, remained there from a previous commit